code/game/objects/items/toys.dm

```diff
@@ -823,6 +853,73 @@
 	desc = "Harvesters tend to have a bad habit of violently stabbing anyone they meet"
 	icon_state = "harvester"
 
+/obj/item/toy/gasha/narnar
+	name = "toy Nar-Sie"
+	desc = "The father figure to all of his faithful, the Geometer of Blood himself; NAR-SIE!"

```

Comment:
narsie is a lady, in olden times she ate the universe with her gaping vagina of fury